src/main/resources/application.conf

```diff
@@ -13,11 +13,17 @@ ats {
     instanceId = ${instanceId}
   }
 }
+
 akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
   loglevel = "DEBUG"
 }
 
+daemon {
+  # example: defaultKeyType = "com.advancedtelematic.libtuf.data.TufDataType.RsaKeyType"
+  defaultKeyType = ${?DEFAULT_KEY_TYPE}
```

Comment:
No default value? Update service.json.

Reply:
I've updated service.json. I didn't put a default value so that the daemon doesn't start. When we need the daemon again we will probably have switched to ed25519.